multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt

```diff
@@ -31,6 +31,7 @@ class MadaraGenerator : ThemeSourceGenerator {
         SingleLang("AZManhwa", "https://azmanhwa.net", "en"),
         SingleLang("Adonis Fansub", "https://manga.adonisfansub.com", "tr", overrideVersionCode = 1),
         SingleLang("Adult Webtoon", "https://adultwebtoon.com/", "en", true),
+        SingleLang("Adult Painful Nightz", "https://adults.painfulnightz.com", "en", true),
```

Comment:
```suggestion
        SingleLang("Adult Painful Nightz", "https://adults.painfulnightz.com", "en",  isNsfw = true),
```

Just to keep it consistent with the other sources.

(You can change the Adult Webtoon one, without bumping the version, too)

Reply:
amended couple of other sources as well since we are linting.